ecosystem/sep-TBD.md

```diff
@@ -0,0 +1,154 @@
+
+#### Using SEP-1
+Many of the [SEP-1][sep1] are highly relevant to NFTs. You should include as many of them as is appropriate for your use case:
+
+  * the `code` and `issuer` fields are essential to describe the Stellar asset that represents your NFT
+  * the `name` and `desc` fields provide human-readable information about your NFT
+  * the `fixed_number`, `max_number`, and `is_unlimited` fields are mutually exclusive ways to describe the supply of your NFT
+  * the `display_decimals` field helps render the right quantity of your NFT, since a single unit of it is likely to be represented by one stroop (i.e. "0.0000001") but you'd want to see "1"
+  * the `image` field is how the ecosystem will "draw" your NFT
+
+Note that even if your NFT isn't an image, you may still want to provide a way to represent it as one (like a logo or symbol) to make it stand out. It's good practice to provide an optimized version of the art to allow fast-loading from services and wallets on Stellar.
+
+Here is an example stellar.toml file describing an asset representing a unique digital image, mimicking the JSON metadata file we described [earlier](#describing-your-nft):
+
+```toml
+[DOCUMENTATION]
+ORG_URL="<https://ink.litemint.store>"
+
+[[CURRENCIES]]
+issuer="GAKZGD5BFXZ7P7P45WHTM6DODMOEWWJUSCAIPGYIVIPYQSVR6MM6YR7M"
+code="DEMOASSET"
+name="A demonstration of NFT metadata"
+desc="This is a description of the cool NFT used in this demo."
+image="https://cloudflare-ipfs.com/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/I/m/RickAstleyNeverGonnaGiveYouUp7InchSingleCover.jpg"
+fixed_number=1
+display_decimals=7
+```
+
+Once you have a `stellar.toml` file under your domain, you should also configure the issuing account's `homedomain` to point to it via the [`SetOptions` operation][set-options]. This unifies the three separate components: the account issuing the NFT, the metadata describing the NFT, and the NFT itself.
+
+#### SEP-1 Extensions
+Since NFTs can represent anything, but the SEP-1 currency specification does not have a generic way to refer to an "anything." There is an `image` key, but it is generally used for something like a currency logo.
+
+Thus, this SEP adds additional supported fields to the SEP-1 currency specification to faciliate this need:
+
+| Field Name | Description and purpose |
+|------------|-------------------------|
+| `url`      | A [valid URL][url] pointing to a NFT that this asset represents |
+| `urlhash`  | A [SHA-256][sha2] hash of the data pointed to by `url` |
```

Comment:
Is anyone using `urlhash` today in SEP-1?

Reply:
These are both _new_ fields introduced in this SEP to extend SEP-1.